website/versioned_docs/version-5.x/layout-definition.md

```diff
@@ -162,20 +162,29 @@ The same application may appear multiple times in your layout, under different r
 <application name="appName" props="myProp,authToken"></application>
 ```
 
+#### application config using JSON
+
 ```js
 // Basic usage
 {
   "type": "application",
   "name": "appName"
 }
 
+import singleSpaReactParcel from 'single-spa-react'
```

Comment:
The name of this variable is misleading, since single-spa-react creates an object that can be used as either an application or parcel. This is why other documentation does this as `import singleSpaReact from 'single-spa-react';`

Reply:
Yeah, I get it. Just took me a little to get my head around. Parcel is just a watered down app, essentially - a "partial".